data.py

```diff
@@ -0,0 +1,66 @@
+import pymongo
+import pprint
+
+from pandas import DataFrame
+
+from dash.exceptions import PreventUpdate
+
+class MongoManager():
+    def __init__(self, server='localhost', port='27017'):
+        self._client = pymongo.MongoClient(server, int(port))
+
+    def _get_database_names(self, ):
+        '''Populate dropdown with database on MongoDB/'''
+        dbl = self._client.list_database_names()
+        return dbl
+
+    def _init_connection_to_runs(self, db_name):
+        '''Generic function to access directly to runs collection of selected database'''
+        db = self._client[db_name]
+        return db['runs']
+
+    def _on_load_database_options(self):
```

Comment:
This function can be removed, as it does exactly the same thing as `_get_database_names`